README.md

```diff
@@ -47,7 +47,8 @@
 - Form for creating post and comments.
 - Feature to like a post.
 - Eager loading.
-- Integration tests.
+- Integration specs for Views and fixing n+1 problems.
```

Comment:
Kindly consider adding your coding partner as an author to the readme file authors section because this is a pair programming project and the contributed to your project